resources/views/components/menu/main.blade.php

```diff
@@ -121,7 +121,7 @@
             <x-dropdown-item :href="url('setRequestList.php')">Most Requested</x-dropdown-item>
             <x-dropdown-item :href="route('claims.completed')">New Sets & Revisions</x-dropdown-item>
             <x-dropdown-item :href="route('claims.active')">Sets in Progress</x-dropdown-item>
-            <x-dropdown-item :href="route('game.random')">Random Set</x-dropdown-item>
+            <x-dropdown-item :href="route('game.random')">Random Game</x-dropdown-item>
```

Comment:
> There is one functional change. The "Random Set" label has been changed to "Random Game". IMO, this better describes what the logic is actually doing - redirecting the user to a game page. 

The idea behind using "Random Set" is that it only returns games _with_ achievements.